modules/ROOT/partials/configuration/ai_shortcuts.adoc

```diff
@@ -0,0 +1,91 @@
+== `ai_shortcuts`
+
+The `+ai_shortcuts+` option controls the list of AI Assistant shortcuts available in the `+AI Shortcuts+` toolbar button and menu item.
+
+This option can be configured with an array to present a customised set of AI Assistant shortcuts.
+
+As well, it can be set to a `+Boolean+` value to control the use of the default list of AI Assistant shortcuts.
+
+When not specified, or set to `+true+`, the AI Assistant shortcuts toolbar button and menu item present and display the default set of shortcuts included with the AI Assistant.
+
+When set to `+[]+` (an empty array), or `+false+`, the AI Assistant shortcuts toolbar button and menu item do not present in the {productname} instance.
+
+When configured with an instance-specific object array, the AI Assistant shortcuts toolbar button and menu item present, and display the configured shortcuts when activated.
+
+*Type:* `+Array+` of `+Objects+`, or `+Boolean+`
+
+*Default value:* 
+[source, js]
+----
+[
+  { title: 'Summarize', prompt: 'Briefly summarize this.' },
+  { title: 'Explain', prompt: 'Give a short explanation of this.' },
+  { title: 'Improve grammar', prompt: 'Correct grammar.' },
+  { title: 'Improve writing', prompt: 'Enhance this text.' },
+  { title: 'Simplify language', prompt: 'Simplify this.' },
+  { title: 'Make longer', prompt: 'Expand this.' },
+  { title: 'Make shorter', prompt: 'Condense this.' },
+  { title: 'Change tone', subprompts: [
+    { title: 'Descriptive', prompt: 'Make this descriptive.' },
+    { title: 'Formal', prompt: 'Make this formal.' },
+    { title: 'Casual', prompt: 'Make this casual.' },
+    { title: 'Humour', prompt: 'Add humour to this.' },
+    { title: 'Inspirational', prompt: 'Inspire reader with this.' },
+    { title: 'Persuasive', prompt: 'Make this persuasive.' },
+  ] },
+  { title: 'Change style', subprompts: [
+    { title: 'Business', prompt: 'Make this business-like.' },
+    { title: 'Legal', prompt: 'Legal style this.' },
+    { title: 'Journalism', prompt: 'Journalistic style this.' },
+    { title: 'Medical', prompt: 'Medical style this.' },
+    { title: 'Poetic', prompt: 'Make this poetic.' },
+  ] },
+]
+----
+
+NOTE: The default shortcuts are subject to change in an upcoming release. If you intend to keep these shortcuts, include them within your integration.
+
+
+=== Example: using `ai_shortcuts` to present a customised set of AI Assistant shortcuts
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  ai_shortcuts: [
+    { title: 'Classical', subprompts: 
+      [
+      { title: 'Dialogue', prompt: 'Convert this to a Socratic dialogue.' },
+      { title: 'Homeric', prompt: 'Convert this to a Classical Epic.' },
+      ]
+    },
+    { title: 'Celtic', subprompts:
+      [
+      { title: 'Bardic', prompt: 'Convert this to Bardic verse.' },
+      { title: 'Filí', prompt: 'Convert this to Filí-an verse.' },
+      ]
+    },
+  ]
+});
+----
```

Comment:
Should we include a example of a `ai_shortcuts` in this, that is not classified as a `subprompt`?

Reply:
> Should we include a example of a ai_shortcuts in this, that is not classified as a subprompt?

[Works for me](https://github.com/tinymce/tinymce-docs/commit/65b433bf1b21092999dec75b83c3bb5bea132511).